packages/block-editor/src/hooks/layout.scss

```diff
@@ -39,3 +39,16 @@
 .block-editor-hooks__toggle-control.block-editor-hooks__toggle-control {
 	margin-bottom: $grid-unit-20;
 }
+
+.block-editor-grid-visualizer .components-popover__content * {
+	pointer-events: none !important;
+}
+
+.block-editor-grid-visualizer__grid {
+	display: grid;
+}
+
+.block-editor-grid-visualizer__item {
+	border: 1px dashed $gray-300;
```

Comment:
This will not be visible when the theme background is greyish. I'd go with a `color-mix(in srgb, var(--wp-admin-theme-color) 20%, transparent);` or so.